Plugin/src/main/java/xyz/kyngs/librelogin/bungeecord/BungeeCordLibreLogin.java

```diff
@@ -250,4 +254,13 @@ protected LibraryManager provideLibraryManager() {
         return new BungeeLibraryManager(bootstrap);
     }
 
+    @Nullable
+    @Override
+    public LimboIntegration<ServerInfo> getLimboIntegration() {
+        if (pluginPresent("NanoLimboBungee")) {
+            limboIntegration = new BungeeNanoLimboIntegration(bootstrap.getProxy().getPluginManager().getPlugin("NanoLimboBungee").getClass().getClassLoader(),
+                    getConfiguration().get(ConfigurationKeys.LIMBO_PORT_RANGE));
+        }
+        return limboIntegration;
+    }
```

Comment:
I'm not sure whether lazy-loading the integration is a good idea. If I understand correctly, you said that you cannot load the integration before super.enable() due to some configuration issues. Why can't you load it in the overridden enable block after the super.enable() call?

Reply:
> We must do this because the super.enable() method registers all limbo servers, and if we register the limboIntegration variable after super.enable(), the console will display the warning "Lobby server/world serverNameHere not found!", and the server will not be registered.

I see, well maybe we should add some null-check like `if (limboIntegration == null && pluginPresent("NanoLimboBungee")` so that we do not create a new instance everytime the method's called

Reply:
https://github.com/kyngs/LibreLogin/pull/122/commits/28b98a2720d9131006e4dd65b223d49c19a11dcf